src/client/util/http.cc

```diff
@@ -336,14 +343,20 @@ void HTTP::PrepareI2PRequest() {
 bool HTTP::ProcessI2PResponse() {
   std::string http_version;
   std::uint16_t response_code = 0;
+  Response response;
   m_Response >> http_version;
   m_Response >> response_code;
-  if (response_code == static_cast<std::uint16_t>(ResponseCode::HTTP_OK)) {
+
+  namespace http = boost::network::http;
+  if (response_code == http::basic_response<
+        http::tags::http_server>::status_type::ok){
     bool is_chunked = false;
     std::string header, status_message;
     std::getline(m_Response, status_message);
     // Read response until end of header (new line)
     while (std::getline(m_Response, header) && header != "\r") {
+      //TODO(aj) if response object none of this is needed.
```

Comment:
Use consistent name for TODO's (either guzzi or aj, not both). Also, two spaces after comments and `TODO(name):` as noted in our style guide.